actix-files/src/files.rs

```diff
@@ -106,7 +106,7 @@ impl Files {
         };
 
         Files {
-            path: mount_path.to_owned(),
+            path: mount_path.trim_end_matches('/').to_owned(),
```

Comment:
Many tests in `files` failed because they use `File::new("/", _)`. A similar thing needs to be done for `Scope` if we want to keep this behavior.

Reply:
it's definitely reasonable to support this format where we're defining a service at the root path

Reply:
yes, trimming a trailing slash would perfectly emulate the old behavior.  note that scope is arguably already broken by the update to router 0.5: `scope("/").route("/")` wouldn't match the root path.

Reply:
Ideally I want the users to use empty prefixes but that is not what most users expect. So the solution I propose is to just "normalize" prefix pattern (by removing trailing slash). This way we get the benefit of greatly simplifying ResourceDef logic and not breaking user expectations.

The question is whether to normalize it in acix-web or in ResourceDef constructors?